app/controllers/v1/workflow/processes_controller.rb

```diff
@@ -0,0 +1,8 @@
+class V1::Workflow::ProcessesController < ApiController
+  def show
+    # TODO: identify current user, check if process id is accessible to user
+    @process = Workflow::Instance::Process.find_by(external_identifier: params[:id])
```

Comment:
`find_by!` w/ the exclamation will error out on a bad ID and return a 404 automatically; just slightly better error handling if you do that vs getting nil error.